dataland-frontend/src/utils/ValidationUtils.ts

```diff
@@ -68,3 +58,31 @@ export function isCompanyIdValid(companyId: string): boolean {
   const uuidRegexExp = /^[0-9a-f]{8}-[0-9a-f]{4}-[0-5][0-9a-f]{3}-[089ab][0-9a-f]{3}-[0-9a-f]{12}$/i;
   return uuidRegexExp.test(companyId);
 }
+
+export const regexPageNumber = /^([1-9]\d*)(?:-([1-9]\d*))?$/;
+
+export const PAGE_NUMBER_VALIDATION_ERROR_MESSAGE =
+  'Page number must be a positive number or ' + 'a range of ascending positive numbers, e.g. 2, 13-15 etc.';
```

Comment:
Is there a reason why this is not one string, but two concatenated strings?

Reply:
The linewidth limitation

Reply:
Ah sorry, this was only an issue in a previous, longer version of the string.  I will fix it.